### PR TITLE
Fix/removedefaultkeys

### PR DESCRIPTION
**Pull request by kwendel**
Remove the defaultkeys from graphstream. This is usefull since we use our own shortkeys

### CODE REVIEW

**Comment by kwendel** [comment]
Zoom in/out with mouse is in the commit history due to a merge error from me. 
But this is not added with this pull request!

**Comment by Delfil** [comment]
All ok

File: src/main/java/tudelft/ti2806/pl3/Application.java [hunk 1]
```diff
@@ -113,6 +113,7 @@ public void makeGraph() {
 			KeyController keys = new KeyController(this);
 			graphView.getPanel().addKeyListener(keys);
 
+
```
Comment (TomBrouws) [inline src/main/java/tudelft/ti2806/pl3/Application.java]: Y u do dis???

File: src/main/java/tudelft/ti2806/pl3/visualization/GraphView.java [hunk 2]
```diff
@@ -111,8 +112,18 @@ private void generateViewer() {
...
 		panel = viewer.addDefaultView(false);
+		removeDefaultKeys();
 	}
-	
+
+	/**
+	 * Remove the default keys from the GraphStream library, since we use our own.
+	 * There is no other way than this to do it.
+	 */
+	public void removeDefaultKeys(){
```
Comment (TomBrouws) [inline src/main/java/tudelft/ti2806/pl3/visualization/GraphView.java]: { is not preceded by whitespace (checkstyle)

File: src/main/java/tudelft/ti2806/pl3/visualization/GraphView.java [hunk 3]
```diff
@@ -142,7 +153,7 @@ public Graph generateGraph() {
 					Node graphNode = graph.addNode(node.getIdString());
 					double y = node.getY() * someSize;
 					graphNode.setAttribute("xy", node.getX(), y);
-					graphNode.addAttribute("ui.class", node.getOriginalNode().getClass()
+					graphNode.addAttribute("ui.class", node.getClass()
```
Comment (mathieupost) [inline src/main/java/tudelft/ti2806/pl3/visualization/GraphView.java]: Y u do dis?
Schiet helemaaal niet op zo

**Comment by mathieupost** [comment]
Please revert the change in GraphView.
Rest seems fine.

**Comment by borismattijssen** [comment]
Looking good. Needs rebase tho ;)

**Comment by TomBrouws** [comment]
Rebase is never breder When meeging to master! Only when you want master commits in your branch to work on

**Comment by TomBrouws** [comment]
Never needed when merging*

**Pull request merged**